aries-site/src/pages/components/data.mdx

```diff
@@ -0,0 +1,162 @@
+import { Example } from '../../layouts';
+import {
+  DataExample,
+  DataFilterExample,
+  DataFiltersExample,
+  DataSearchExample,
+  DataSortExample,
+  DataSummaryExample,
+  DataTableColumnsExample,
+  DataTableGroupByExample,
+  DataViewExample,
+  ToolbarExample,
+} from '../../examples/components/';
+
+Data is the wrapper component and orchestrator for a set of subcomponents. Because this set of components work together to create the entirety of the experience, we like to refer to them as "Data and friends".
+
+<Example
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/data/DataSearchExample.js"
+  docs="https://v2.grommet.io/data"
+  template
+  height="auto"
+  showResponsiveControls={false}
+>
+  <DataExample />
```

Comment:
The pagination for this example says "showing 1-10 of 21 items", but more than 10 items are visible on the first page.

<img width="925" alt="Screen Shot 2024-02-09 at 1 25 55 PM" src="https://github.com/grommet/hpe-design-system/assets/54560994/a51bfd0d-3538-4faf-8d85-88d9c1840aa1">

Reply:
This is fixed now. It seems to be something with when `views` is defined. I've removed it for now and will file a grommet ticket to investigate separately.

Reply:
Another similar issue, when I search and end up with 2 results the pagination shows that it is showing '1-10' items. We can look into separately from this PR though.

<img width="728" alt="Screen Shot 2024-02-12 at 3 41 21 PM" src="https://github.com/grommet/hpe-design-system/assets/54560994/709180f4-b0b9-422f-a89f-62257db5dc07">